apps/ledger-live-desktop/package.json

```diff
@@ -144,6 +144,7 @@
     "winston": "^3.2.1",
     "winston-transport": "^4.3.0",
     "write-file-atomic": "^5.0.0",
+    "xrpl-binary-codec-prerelease": "^8.0.1",
```

Comment:
I don't see any reference to this in `ledger-live-desktop` project, why was it added ?

Reply:
It is a replacement of `ripple-binary-codec`, where `ripple-binary-codec` only supports the native XRPL chains, with fixed network definitions: so whenever a new transaction type, field or amendment is introduced, it needs an update.

This package (`xrpl-binary-codec-prerelease`) implements dynamic network definitions as served by both XRPL and other similar protocol chains over RPC, so it can always deal with new TX types fields & netwoks.

The changes to make this possible have been PR'ed into `ripple-binary-codec` but haven't been merged for ages. Several projects now use `xrpl-binary-codec-prerelease`.